packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -154,14 +172,39 @@ const Tab = ({
 
   if (isIconOnly) {
     attributes['aria-label'] = label;
+    if (isClosable) {
+      attributes['aria-label'] = `${label} ${tabDeleteLabel}`;
```

Comment:
we need to add condition here to add label only when it has valid value to prevent screen readers from reading undefined OR space OR null .

Reply:
Thanks Added https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41